static/templates/bot_avatar_row.handlebars

```diff
@@ -23,6 +23,10 @@
             <div class="field">{{t "Username" }}</div>
             <div class="value">{{email}}</div>
         </div>
+        <div class="type">
+            <div class="field">{{t "Bot type" }}</div>
+            <div class="value">{{t type }}</div>
```

Comment:
The if statements don't work  well for comparison operators in handlebars files (I'll have to add multiple nested if statements for the same).
That is why I preferred this. Is this not the correct way to do it?